.gitignore

```diff
@@ -9,3 +9,4 @@ phpunit.phar
 /tests/docker
 /tests/dockerids
 
+/.idea
```

Comment:
this should go into your gobal gitignore http://stackoverflow.com/questions/7335420/global-git-ignore#7335487